views/404.ejs

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title><%-data.name%></title>
+    <style>
+    * {
```

Comment:
Actually you should be able to use includes for this, see https://github.com/tj/ejs#includes